script/windows-installer/inno-setup-git-lfs-installer.iss

```diff
@@ -45,6 +45,7 @@ PrivilegesRequired=none
 SetupIconFile=git-lfs-logo.ico
 SolidCompression=yes
 UsePreviousAppDir=no
+UninstallDisplayIcon={app}\git-lfs.exe
```

Comment:
Sorry to be a nitpicker here, but that's still not sorted alphabetically, as it should come *before* `UsePreviousAppDir` :wink:

Reply:
@sschuberth not at all - my bad!